docs/dashboard/catalog/products.md

```diff
@@ -286,41 +286,59 @@ Based on these attributes, you can create variants of your product.
 
-![Product Variant Creator](../screenshots/catalog-product-variant-attributes.jpeg)
+:::
 
-#### Price Section
+#### 2. Prices and SKU tab
 
-Specify how you want your prices to be applied to each variant:
+##### Price section
+The first step is to specify how you want your prices to be applied to each variant:
 
-- Apply single price to all SKUs - All attribute values of this product to have the same price.
+- Apply single price to all SKUs for each channel - All attribute values of this product to have the same price.
 - Apply unique prices by attribute to each SKU - Search attribute has a different value differently which you must manually assign.
+- Skip pricing for now
 
-![Product Variant Creator](../screenshots/catalog-product-variant-pricing.jpeg)
+![Product Variant Creator](../screenshots/catalog-product-variant-pricing.png)
 
-#### Stock section
+##### Stock and Warehousing section 
+After filling out the price section, you need to assign a previously created [warehouse](dashboard/configuration/warehouses.mdx):
 
-How you want to manage your stock:
+:::note
+You can assign multiple warehouses.
+:::
+
+Once specific warehouses have been selected, you can manage your stock:
 
 - Apply single stock to all SKUs - Select this option if you have the same amount of each product.
 - Apply unique stock by attribute to each SKU - Use this option if your stock levels are different for each variant.
+- Skip stock for now
+
+#### 3. Summary tab
 
-Click _Next_ to progress to the Summary tab containing an overview of the variants you’ve just created. You can still edit inventory, price and SKU.
+Click _Next_ to progress to the Summary tab containing an overview of the variants you’ve just created. You can still edit price per channel and SKU.
 
-![Product Variant Creator](../screenshots/catalog-product-variant-stock.jpeg)
+![Product Variant Creator](../screenshots/catalog-product-variant-stock.png)
 
 Once saved, you can edit variants through the same steps as described above and then save changes.
 
-#### Checking inventory
+### Checking inventory
 
-In this example, you can see five types of t-shirt. Four have enough stock to cover future orders. The S size has a red mark and is unavailable. When we click on the product, we can see in the product variant details that the product is unavailable because there are fourteen items in stock (“Inventory”) and they have been ordered by a customer (“Allocated”).
+In this example, when you click on the product, we can see in the product variant details page that the product is still available because there are one hundred thirty eight (138) items in the stock (the _Quantity_ section) and they have been ordered sixty nine (69) times by the customers so far (the _Allocated_ section).
 
-![Checking inventory](../screenshots/catalog-product-stock.jpeg)
+![Checking inventory](../screenshots/catalog-product-checking-inventory.jpg)
```

Comment:
Still `jpg`

Reply:
In this case, I prefer to leave jpg. As far as I can see, there is no such example in our current dashboard (number of allocated products, etc.).